bin/download

```diff
@@ -135,11 +137,15 @@ guess_download_url() {
   declare -r arch="$(get_arch)"
   declare -r platform="$(get_platform)"
 
+  if [ "$arch" == 'aarch64' ]; then
```

Comment:
How can this be true? Isn't `$arch` always `'32'` or `'64'`? I'm sure we're always downloading the intel image.